src/components/ButtonPanel/index.js

```diff
@@ -1,39 +1,47 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
+import Proptypes from 'prop-types';
 import Button from '../Button';
 import './index.css';
+import * as operations from '../../logic/operationalConstants';
 
-const ButtonPanel = () => (
+const ButtonPanel = ({ clickHandler }) => (
   <div className="buttonPanel flex flex-column">
     <div className="group">
-      <Button btnName="AC" color="#dfdfdf" />
-      <Button btnName="+/-" color="#dfdfdf" />
-      <Button btnName="%" color="#dfdfdf" />
-      <Button btnName="÷" />
+      <Button onClick={clickHandler} btnName={operations.CLEAR} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.PLUS_NEGATIVE} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.PERCENTAGE} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.DIVIDE} />
     </div>
     <div className="group">
-      <Button btnName="7" color="#dfdfdf" />
-      <Button btnName="8" color="#dfdfdf" />
-      <Button btnName="9" color="#dfdfdf" />
-      <Button btnName="X" />
+      <Button onClick={clickHandler} btnName={operations.SEVEN} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.EIGHT} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.NINE} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.MULTIPLY} />
     </div>
     <div className="group">
-      <Button btnName="4" color="#dfdfdf" />
-      <Button btnName="5" color="#dfdfdf" />
-      <Button btnName="6" color="#dfdfdf" />
-      <Button btnName="-" />
+      <Button onClick={clickHandler} btnName={operations.FOUR} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.FIVE} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.SIX} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.SUBTRACT} />
     </div>
     <div className="group">
-      <Button btnName="1" color="#dfdfdf" />
-      <Button btnName="2" color="#dfdfdf" />
-      <Button btnName="3" color="#dfdfdf" />
-      <Button btnName="+" />
+      <Button onClick={clickHandler} btnName={operations.ONE} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.TWO} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.THREE} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.SUM} />
     </div>
     <div className="group">
-      <Button btnName="0" color="#dfdfdf" wide="true" />
-      <Button btnName="." color="#dfdfdf" />
-      <Button btnName="=" />
+      <Button onClick={clickHandler} btnName={operations.ZERO} color="#dfdfdf" wide="true" />
+      <Button onClick={clickHandler} btnName={operations.DOT} color="#dfdfdf" />
+      <Button onClick={clickHandler} btnName={operations.EQUAL} />
```

Comment:
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.